front/app/components/HookForm/Feedback/index.tsx

```diff
@@ -57,7 +57,7 @@ const Feedback = ({
 
         errorMessages.push({
           field,
-          message: formatMessage(apiErrorMessage),
+          message: apiErrorMessage ? formatMessage(apiErrorMessage) : '',
```

Comment:
Allowing for an empty string here to account for error messages that have not been mapped to error copy in the same way this component expects (like with the custom fields).